centreon/www/install/php/Update-22.04.14.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+require_once __DIR__ . '/../../class/centreonLog.class.php';
+$centreonLog = new CentreonLog();
+
+//error specific content
+$versionOfTheUpgrade = 'UPGRADE - 22.04.14: ';
+$errorMessage = '';
+
+//Change the type of check_attempt and max_check_attempts columns from table resources
+$errorMessage = "Couldn't modify resources table";
+$alterResourceTableStmnt = "ALTER TABLE resources MODIFY check_attempts SMALLINT UNSIGNED, 
+    MODIFY max_check_attempts SMALLINT UNSIGNED";
+
+try {
+
+    $pearDBO->query($alterResourceTableStmnt);
+    $errorMessage = '';
```

Comment:
Empty error message ?

Reply:
i reset the variable just after the query  to avoid false positives in future